### PR TITLE
GH-827 Adds Amchart Balloon (Tooltip)

### PR DESCRIPTION
**Pull request by mlzummo**
@GordonSmith @jbrundage @dtsnell4 Please Review

Adds AMCHART BALLOON with same output formatting as HPCC Charts

Fixes GH-827

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
http://rawgit.com/mlzummo/Visualization/GH_827_AMCHART_BALLOON/demos/dermatology.html

File: src/amchart/Pie.js [hunk 1]
```diff
@@ -65,6 +63,8 @@
         this._chart.fontSize = this.fontSize();
         this._chart.color = this.fontColor();
 
+
+        this._chart.outlineColor = "#000000";
```
Comment (jbrundage) [inline src/amchart/Pie.js]: This should probably be a publish parameter

File: src/common/HTMLWidget.js [hunk 2]
```diff
@@ -187,4 +187,4 @@
     };
 
     return HTMLWidget;
-}));
+}));
```
Comment (jbrundage) [inline src/common/HTMLWidget.js]: You should remove this change

File: src/amchart/Pie.js (hunk 1 above)
Comment (mlzummo) [inline src/amchart/Pie.js]: thats not suppose to be there good find

**Comment by mlzummo** [comment]
closing in favor of https://github.com/hpcc-systems/Visualization/pull/853

**Pull request closed without merging**